lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -172,8 +183,6 @@ export function createBucket(authInfo, bucketName, headers,
 
     if (locationConstraint !== undefined) {
         bucket.setLocationConstraint(locationConstraint);
-    } else if (usEastBehavior) {
-        bucket.setLocationConstraint('us-east-1');
```

Comment:
what do we do if someone has usEastBehavior set but does not send a location on bucket put?